src/algebra/group_with_zero/units.lean

```diff
@@ -97,6 +97,16 @@ by rw [← mul_assoc, mul_inverse_cancel x h, one_mul]
 lemma inverse_mul_cancel_left (x y : M₀) (h : is_unit x) : inverse x * (x * y) = y :=
 by rw [← mul_assoc, inverse_mul_cancel x h, one_mul]
 
+lemma inverse_mul_eq_iff_eq_mul (x y z : M₀) (h : is_unit x) :
+  ring.inverse x * y = z ↔ y = x * z :=
+⟨λ h1, by rw [← h1, ring.mul_inverse_cancel_left _ _ h],
+  λ h1, by rw [h1, ring.inverse_mul_cancel_left _ _ h]⟩
+
+lemma eq_mul_inverse_iff_mul_eq (x y z : M₀) (h : is_unit z) :
+  x = y * ring.inverse z ↔ x * z = y :=
+⟨λ h1, by rw [h1, ring.inverse_mul_cancel_right _ _ h],
+  λ h1, by rw [← h1, ring.mul_inverse_cancel_right _ _ h]⟩
```

Comment:
You are in the `ring` namespace, you shouldn't need to write `ring.`

Reply:
Good point, thanks!